templates/monorepo/packages/utils/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "@mono/utils",
+  "version": "0.0.0",
```

```suggestion
  "version": "0.0.0",
  "license": "MIT",
```

MIT license missing. We probably want to include a `LICENSE` file in each package as well. NPM will automatically include any `LICENSE` file when publishing (similar to `README.md`)